app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import shlex
```

Comment:
I think no need here to use this library
It is useful if names of file have blanks, but in your test case you can just use simple split